### PR TITLE
Fixes #34806 - Replace pf3 OverlayTrigger

### PR DESCRIPTION
**Pull request by MariaAga**

### CODE REVIEW

File: webpack/ForemanTasks/Components/TasksDashboard/Components/TasksCardsGrid/Components/StoppedTasksCard/OtherInfo.js [hunk 1]
```diff
@@ -2,38 +2,35 @@ import React from 'react';
...
+      <Tooltip
+        id="stopped-tooltip"
+        content={__(
+          'Other includes all stopped tasks that are cancelled or pending'
+        )}
+        position={TooltipPosition.bottom}
       >
         <span>
-          <Icon type="pf" name="info" />
+          <InfoCircleIcon className="pficon" />
```
Comment (MariaAga) [inline webpack/ForemanTasks/Components/TasksDashboard/Components/TasksCardsGrid/Components/StoppedTasksCard/OtherInfo.js:28]: Using the same classname as pf3 so the css we have for other icons in the page will apply

**Comment by adamruzicka** [comment]
Probably unrelated to this PR per-se, but something odd I noticed. Sometimes (I think there is a click involved), the popups don't go away and you can end up with something like this 

image

Apart from this it seems to work just fine

**Comment by MariaAga** [comment]
These tooltips come from the browser by using `title=...` in the html so that probably unrelated.
Also tried to reproduce it on chrome and firefox and couldn't

**Review by ofedoren — COMMENTED** [review]
Thanks, @MariaAga! Nice change. The corresponding changes were merged in Foreman, so I guess we can merge this as well. The only thing I'd like to change is the issue ticket to be in ForemanTasks component rather than Foreman and just linked them, but that's just a nitpick...

Also, I've noticed a slight difference in alignment: the number of the "other" tasks is kinda off. Otherwise looks better :)

Before:
before
After:
after

**Comment by MariaAga** [comment]
Thanks @ofedoren!
the number got a bit bigger as pf4 font size is 2px larger than pf3 so it looks different

**Review by ofedoren — APPROVED** [review]
I don't see any other reason not to get it in, thanks @MariaAga :)

P.S. PR's title differs from the commit name.

**Review by ezr-ondrej — APPROVED** [review]
Thanks @MariaAga and @ofedoren ! :+1:

**Pull request merged**